Resources/Prototypes/Entities/Structures/Specific/bluespace_harvster_rift.yml

```diff
@@ -0,0 +1,102 @@
+- type: entity
+  id: BluespaceHarvesterRift
+  name: redspace rift
+  description: Run! Run while you can! Just run!
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: BluespaceHarvesterRift
+    passiveSpawn:
+    - MobFleshJared
+    - MobFleshGolem
+    - MobFleshClamp
+    - MobFleshLover
+    - FleshBlocker
+    spawn:
+      # Flesh
+      - id: MobFleshJared
+        cost: 30
+      - id: MobFleshGolem
+        cost: 30
+      - id: MobFleshClamp
+        cost: 30
+      - id: MobFleshLover
+        cost: 30
+      - id: FleshBlocker
+        cost: 30
+      # Slimes
+      - id: MobAdultSlimesBlueAngry
+        cost: 40
+      - id: MobAdultSlimesYellowAngry
+        cost: 40
+      - id: MobAdultSlimesGreenAngry
+        cost: 40
+      - id: ReagentSlimeSpawner
+        cost: 60
+      # Carps
+      - id: MobCarp
+        cost: 70
+      - id: MobCarpMagic
+        cost: 90
+      # Space
+      - id: MobKangarooSpace
+        cost: 35
+      - id: MobBearSpace
+        cost: 35
+      - id: MobPurpleSnake
+        cost: 35
+      - id: MobTick
+        cost: 35
+      - id: MobWatcherLavaland
+        cost: 100
+      # Spiders
+      - id: MobSpiderSpace
+        cost: 50
+      - id: MobGiantSpiderAngry
+        cost: 55
+      - id: MobClownSpider
```

Comment:
I don't particularly like that all the monster types and categories are mixed into one big pile. I'd like to see some sort of monster categorization based on the current type of harvester operation, but that's wyci